guides/v2.2/config-guide/cli/config-cli-subcommands-cache.md

```diff
@@ -111,16 +111,23 @@ To view the status of the cache, enter
 
 A sample follows:
 
+There are 13 types of cache types display as result for Magento open source by default.
+
```

Comment:
Merge with "A sample follows:" Change the colon to a period. Then, say "Enabled cache types have a value of 1. Otherwise, the value is 0."